llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -2089,6 +2087,7 @@ class MUBUF_Real_gfx11<bits<8> op, MUBUF_Pseudo ps,
   let Inst{53}    = ps.tfe;
   let Inst{54}    = ps.offen;
   let Inst{55}    = ps.idxen;
+  let SubtargetPredicate = isGFX11Only;
```

Comment:
Instead of this, could you put something in a base class so that all Reals copy their SubtargetPredicate from the corresponding Pseudo?